Sm/source tracking (#272)
salesforcecli/plugin-deploy-retrieve: messages/deploy.metadata.md

You must run this command from within a project.

> This command uses source-tracking if support on the org, unless you set the org to not track source when it was created.

Reviewer: ```suggestion
If your org allows source tracking, then this command tracks the local and remote changes in your source. Some orgs, such as production org, never allow source tracking. You can also use the "--no-track-source" flag when you create a scratch or sandbox org to disable source tracking.
```

Validate deploy and run Apex tests but don’t save to the org.

# flags.ignore-conflicts.summary

> For orgs that use source tracking, deploy even if it will overwrite changes in the org.

Reviewer: ```suggestion
Ignore conflicts and deploy local files, even if they overwrite changes in the org.
```

# warning.TargetOrgIsExpired

The target-org, "%s", is expired. Do you want to pick another org?

# error.Conflicts

> There are changes on the server that conflict with what you're trying to deploy

Reviewer: ```suggestion
There are changes in the org that conflict with the local changes you're trying to deploy.
```

# error.Conflicts.Actions

- Re-run the command with --ignore-conflicts if you want to overwrite the remote changes

> - Retrieve the changes with --ignore-conflicts if you want to overwrite the local changes

Reviewer: ```suggestion
- To overwrite the local changes, run the "sf retrieve metadata" command with the --ignore-conflicts flag.
```
